contracts/test/CompoundGovernor.t.sol

```diff
@@ -736,3 +769,190 @@ contract CastVoteWithReasonAndParams is CompoundGovernorTest {
         governor.castVoteWithReasonAndParams(_proposalId, VOTE_TYPE_FRACTIONAL, "MyReason", _newParams);
     }
 }
+
+contract ProposalDeadline is CompoundGovernorTest {
+    function testFuzz_ProposalDeadlineCorrectWithEnumeratedId(uint256 _proposerIndex) public {
+        _proposerIndex = bound(_proposerIndex, 0, _majorDelegates.length - 1);
+        address _proposer = _majorDelegates[_proposerIndex];
```

Comment:
We can use the existing helper, removing the need for `_proposerIndex`.
```suggestion
        address _proposer = _getRandomProposer();
```

Reply:
Done: 9c1f15694fdc5e64ff58f639c67dd6f72d3e45dd